TESTING.md

```diff
@@ -0,0 +1,79 @@
+# Automated End-to-End Testing
+
+An overview of the End-to-End testing of CHT-Core. 
+
+## Stack overview
+
+### Requirements
+Follow the guide [DEVELOPMENT.md](DEVELOPMENT.MD)
+JDK installed for Selenium.
+Docker to run couchdb.
+
+### Local Run
+
+`grunt e2e` installs and runs chromedriver, starts couchdb in docker, pushes the compiled app to couchdb, starts api, starts sentinel, and then runs protractor tests against your local environment. 
+
+### Github Actions Run
+
+The build process compiles our application. Then installs horticulturalist to run the app. This puts us closer to production. Executes `grunt ci-e2e`. Which then installs and runs chromedriver. Runs the protractor tests against the installed app version. Currently there are 3 jobs that execute in the supported node environments.  
+
+## Protractor Tips
+
+### File Structure
+Tests files should represent a feature within the application. Using `describe` to identify the feature and `it` detail the individual functions of the feature. 
```

Comment:
I think this is appropriate guidance for where we're at now on these. Over time, the way the `describe` blocks work will help us to better test conditions and serve as scoping and work with the before/after and setup code that we need for different contexts.